osquery/utils/conversions/windows/windows_time.cpp

```diff
@@ -56,4 +58,40 @@ LONGLONG littleEndianToUnixTime(const std::string& time_data) {
   return last_time;
 }
 
+LONGLONG parseFatTime(const std::string& fat_data) {
```

Comment:
Please add test in https://github.com/osquery/osquery/blob/7a2a3992f910fbab2e2eb20de6c346e4b0a75e9a/osquery/utils/conversions/windows/tests/windows_time.cpp